spec/ralyxa/configuration_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'ralyxa/configuration'
+
+RSpec.describe Ralyxa::Configuration do
+  subject { Ralyxa::Configuration.new }
+
+  describe '#initialize' do
+    it 'sets default configuration values' do
+      expect(subject.validate_requests).to eq(true)
```

Comment:
What are your thoughts on using `be true` instead of `eq`?

Reply:
I don't have an overt preference, just prefer using eq as it works in all circumstances - I can update this if you'd prefer `be true`?

Reply:
Thinking about this more, I feel that `eq` has a better flow when using the expect syntax versus `be` which seems to fit better when using the old `should` syntax:

```ruby
subject.validate_requests.should be true
subject.validate_requests.should eq true

expect(subject.validate_requests).to be true
expect(subject.validate_requests).to eq true
```

Having consulted with my English teacher friend and my wife who has a masters in English (she made me write that :joy:), she has said that in a sentence:
`subject validate requests should be true` - Is correct English, but in context you should use `subject validate requests should equal true` because it's a test in a programming language and you would say `a equals b` not `a should be b`

---

They went on to say that in the context 'equals' means something to us where it wouldn't in the true English language version of this

---

Hope that makes sense? Again, more than happy to change @sjmog 😄

Reply:
Well-reasoned! I think `be` uses object identity, whereas `eq` is more permissive. But I'm super-happy with either.